modules/packages/MatrixMarket.chpl

```diff
@@ -175,8 +175,8 @@ proc mmwrite(const fname:string, mat:[?Dmat] ?T) where mat.domain.rank == 2 {
    var (nrows, poslast) = (-1,-1);
    var n_cols = -1;
 
-   const DmatHighRow = high(Dmat)(1);
-   const DmatHighCol = high(Dmat)(2);
+   const DmatHighRow = high(Dmat)(0);
```

Comment:
TODO (probably for me): We should propagate these changes to the mason package

Reply:
That'd be great if you could take that (and the LinearAlgebraJama.chpl case).